src/ci/docker/run.sh

```diff
@@ -22,7 +22,17 @@ if [ -f "$docker_dir/$image/Dockerfile" ]; then
       hash_key=/tmp/.docker-hash-key.txt
       rm -f "${hash_key}"
       echo $image >> $hash_key
-      find $docker_dir -type f | sort | xargs cat >> $hash_key
+
+      # Look for all source files involves in the COPY command
+      copied_files=/tmp/.docker-copied-files.txt
```

Comment:
Can you `rm -f` the file? This way the hash is consistent even when testing the script locally.